lib/inets/test/httpc_SUITE.erl

```diff
@@ -56,8 +56,10 @@ all() ->
      {group, http_ipv6},
      {group, sim_http},
      {group, http_internal},
-     {group, http_unix_socket},
      {group, https},
+     %% http_unix_socket group creates a persistent connection session in
+     %% httpc_manager for localhost:80 which may affect tests exected afterwards
+     {group, http_unix_socket},
```

Comment:
This change probably belongs to another branch.

Reply:
It is related in a way that `httpc_SUITE:undefined_port` started failing after adjusting normalize behavior in uri_string.
Another question could be: what `httpc_SUITE:undefined_port` is supposed to test at all and if it is formulated properly.

I will have a look once again.